README.md

```diff
@@ -5,3 +5,104 @@
 This repo contains a Python module `testcases` which provides a Python `unittest.TestCase` subclass which can be used to create integration tests of applications by driving Docker Compose.
 
 See the [API documentation](https://github.com/autopilotpattern/testing/blob/master/API.md) for details on the methods made available on the `AutopilotPatternTest` class.
+
+## Test environments
+
+The minimal environment for an application that uses the `testcases` module is:
+
+- a Docker client.
+- a Docker deployment target such as Triton or Docker for Mac.
+- Python 3
+- installing `requirements.txt` via `pip3` (this will include Docker Compose).
```

Comment:
Understandably, you might not be considering that part of the environment. 

There's an implied knowledge that the user understands they have to bring their own tests. Should that be more explicitly stated up top.